playbooks/roles/mysql/tasks/main.yml

```diff
@@ -21,4 +21,12 @@
   when: remove_experimental_mysql
 
 - include: mysql.yml
-  when: (mysql_56_installed.rc == 1) or (remove_experimental_mysql)
\ No newline at end of file
+  when: (mysql_56_installed.rc == 1) or (remove_experimental_mysql)
+
+- name: Create remote root user
```

Comment:
@mrgnr Ok, gotcha. Yes, we agreed that, but only for pro and enterprise tiers I think, this is intended to use the mysql db on the edxapp server for basic tiers?